src/scss/includes/font.scss

```diff
@@ -28,14 +28,15 @@
 
 .icon-pin_geoloc:before {
   content: "\e90d";
+  color: #5c6f84;
 }
 .icon-check:before {
   content: "\e91c";
 }
 .icon-chevron-left:before {
   content: "\e913";
 }
-.icon-chevron-right:before {
+.icon-chevron-right2:before {
```

Comment:
why this change ?

Reply:
when i update icomoon i need to remove the 2 for left, i missed this time